### DISCUSSION TITLE
Success with Low-Wall Console indoor units

### DISCUSSION THREAD

**Discussion by TerryFrench** [JanM321/esphome-lg-controller#66] [Show and tell]
LG AMNW12GQA0 aka LQN120HV4 (Low-Wall Console Indoor Unit) and the 9kbtu siblings AMNW09GQA0 aka LQN090HV4 work very well. Only oddity: airflow up-down is reversed. Outdoor unit is A4UW30GFH0 aka LMU300HHV

It's using the tiny version of the boards, built from JLPCB 

A bit more details there https://github.com/jourdant/esphome-lgap/issues/10#issuecomment-2264531969

**Comment by JanM321** [comment]
Thanks for updating the wiki.

For the airflow setting: I wonder what the official LG controllers do in that case... It's possible they have the same issue or maybe there's an extra flag somewhere that makes them reverse this setting in your case.

If you could post the `C9...`, `CA...` etc messages from the ESPHome log file I could check that when I have time. The `C9` message is sent by the unit right after (re)starting the controller.

**Reply by TerryFrench** [reply]
I will try. Thanks
